i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx

```diff
@@ -467,8 +467,8 @@ vehicles:
     type: template
     template: tesla
     title: # Will be displayed in the user interface (optional)
-    accessToken: # See https://docs.evcc.io/docs/devices/vehicles#tesla
-    refreshToken: # See https://docs.evcc.io/docs/devices/vehicles#tesla
+    accessToken: # See https://docs.evcc.io/en/docs/devices/vehicles#tesla
+    refreshToken: # See https://docs.evcc.io/en/docs/devices/vehicles#tesla
```

Comment:
This is autogenerated. The source is in evcc core. Can you make small PR there?

Reply:
Are you suggesting that it should be a relative link? If so, does that make sense in a comment?

If not, what change are you looking for here?

Reply:
No, we can't use relative links here. The same text is used in `evcc configure` and will be used in the Web Config UI.

The only thing I wanted to say is, that the code you've edited is autogenerated and will be overwritten. The source is here: https://github.com/evcc-io/evcc/blob/master/templates/definition/vehicle/tesla.yaml#L29

Reply:
I've made this change here. There were a bunch of other locations that needed adjustment as well.
https://github.com/evcc-io/evcc/pull/10053